tests/CodeCollection/fixtures/CodeCollection/ChangeApplier.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Upgrader\CodeCollection;
```

Comment:
Instead of using duplicates of the core classes please mock up some dummy classes in the right PSR-4 namespace for tests.